### PR TITLE
Appveyor GitHub releases

### PR DESCRIPTION
**Pull request by fholger**
This change makes AppVeyor create a GitHub draft release whenever a new tag is pushed to the repository. The draft release contains the built ReviveInstaller.exe, but nothing else. The details can be edited (title and description should be adjusted as needed) and then the release can be published. This should potentially make it easier to prepare new releases for any repo maintainer, since builds don't need to be prepared manually :)

By default, when a tag release is being built, the NSIS installer is instructed to create a proper build, i.e. without the initial message box warning about installing a beta release. If the tag name ends in "beta", then the message box will appear.

To make this work, AppVeyor needs a proper GitHub token for the repository. It's stored in an encrypted format (so that only the AppVeyor account can use it) in the appveyor.yml under deploy/auth_token/secure. Follow the instructions at https://www.appveyor.com/docs/deployment/github/ under *GitHub authentication token*.

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -52,10 +52,24 @@ after_build:
...
 artifacts:
   - path: Installer\ReviveInstaller.exe
     name: Revive Installer
   - path: Release\Revive.zip
     name: Revive Build
+
+deploy:
+  provider: GitHub
+  auth_token:
+    secure: 1ckkLEAo2w5lZ4oziHqrXRQvuRwQoLuu9Q0ypuvk9CFp005B6T02JvV6aQdAGrHU
```
Comment (fholger) [inline appveyor.yml]: This needs to be changed to a valid token for the LibreVR/Revive repository (see PR description).

Comment (highb) [reply appveyor.yml:70]: FYI, this token is visible to the public. You might want to use [github's built in secrets feature for the repo](https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets) to hide the token.

Comment (fholger) [reply appveyor.yml:70]: Thanks for your concern. However, this token is already encrypted, see PR description. Unless you have access to the encrypting AppVeyor account, you can't do anything with it.

**Comment by highb** [comment]
Ah, gotcha. Thanks for your work!

**Pull request merged**